testdata/p4_16_samples_outputs/table-entries-exact-ternary-bmv2.p4.entries.txt

```diff
@@ -0,0 +1,124 @@
+updates {
+  type: INSERT
+  entity {
+    table_entry {
+      table_id: 33616996
+      match {
+        field_id: 1
+        exact {
+          value: "\001"
+        }
+      }
+      match {
+        field_id: 2
+        ternary {
+          value: "\021\021"
+          mask: "\000\017"
+        }
+      }
+      action {
+        action {
+          action_id: 16837978
+          params {
+            param_id: 1
+            value: "\000\001"
+          }
+        }
+      }
+      priority: 1
+    }
+  }
+}
+updates {
+  type: INSERT
+  entity {
+    table_entry {
+      table_id: 33616996
+      match {
+        field_id: 1
+        exact {
+          value: "\002"
+        }
+      }
+      match {
+        field_id: 2
+        ternary {
+          value: "\021\201"
+          mask: "\377\377"
+        }
+      }
+      action {
+        action {
+          action_id: 16837978
+          params {
+            param_id: 1
+            value: "\000\002"
+          }
+        }
+      }
+      priority: 2
+    }
+  }
+}
+updates {
+  type: INSERT
+  entity {
+    table_entry {
+      table_id: 33616996
+      match {
+        field_id: 1
+        exact {
+          value: "\003"
+        }
+      }
+      match {
+        field_id: 2
+        ternary {
+          value: "\021\021"
+          mask: "\360\000"
+        }
+      }
+      action {
+        action {
+          action_id: 16837978
+          params {
+            param_id: 1
+            value: "\000\003"
+          }
+        }
+      }
+      priority: 3
+    }
+  }
+}
+updates {
+  type: INSERT
+  entity {
+    table_entry {
+      table_id: 33616996
+      match {
+        field_id: 1
+        exact {
+          value: "\004"
+        }
+      }
+      match {
+        field_id: 2
+        ternary {
+          value: "\000\000"
+          mask: "\000\000"
```

Comment:
Should P4Runtime canonical representation of omitting fields that are completely "don't care" apply in this file?